data/tutorials/getting-started/gs_01_a_tour_of_ocaml.md

```diff
@@ -0,0 +1,874 @@
+---
+id: a-tour-of-ocaml
+title: A Tour of OCaml
+description: >
+  Hop on the OCaml sightseeing bus. This absolute beginner tutorial will drive you through the marvels and wonders of OCaml. We'll have a look at the most commonly used language features.
+category: "Getting Started"
+---
+
+# A Tour of OCaml
+
+> Franz avait lu dans France Soir qu’un Américain avait mis neuf minutes et quarante-cinq secondes pour visiter le musée du Louvre. Ils décidèrent de faire mieux…
+
+&mdash; Jean-Luc Godard, _Bande à part_
+
+Let's walk through the basics of OCaml by trying out different elements in an interactive manner. We recommend that you execute the examples we provide, or slight variants of them, in your own environment to get a feel for coding in OCaml.
+
+## Prerequisites and Goals
+
+Before proceeding with this tutorial, please ensure you've installed OCaml and set up the environment, as described on the [install OCaml](/docs/installing-ocaml) page. After we take an introductory tour of OCaml's language features, we'll proceed to create our first OCaml project in the [How to Write an OCaml Program](/docs/how-to-write-an-ocaml-program) tutorial.
+
+This is a level zero tutorial. No OCaml or functional programming knowledge is required. As mentionned above, only having OCaml installed is needed. It is assumed the reader has some basic software development knowledge. This tutorial is probably not adapted to learn programming.
+
+The goal of this tutorial is to provide the following capabilities:
+- Use Utop to evaluate OCaml expressions interactively
+- Trigger evaluation of expressions, understand the output displayed, values and typing
+- Write definitions of values and functions
+- Write list literals, use basic list operation, mattern match on list values
+- Write simple float expressions without typing errors
+- Write and patch-match pair and tuples values
+- Define a variant types, create and pattern match on values
+- Define an immutable record type, create a values, access the fields
+- Raise and catch an exception
+- Return an error-as-data result, process it using pattern matching
+- Declare and update a mutable basic mutable values: references and arrays
+- Call functions defined in modules of the OCaml standar library
```

Comment:
```suggestion
Before proceeding with this tutorial, please ensure you've installed OCaml and set up the environment, as described on the [install OCaml](/docs/installing-ocaml) page. After we take an introductory tour of OCaml's language features, we'll proceed to create our first OCaml project in the [How to Write an OCaml Program](/docs/how-to-write-an-ocaml-program) tutorial.

This is a level zero tutorial. No OCaml or functional programming knowledge is required. Although, it is assumed the reader has some basic software development knowledge. 

This document will cover how to use the REPL UTop to evaluate OCaml expressions interactively, understand the output, how to use pattern matching, call functions in OCaml standard library modules, and more. It also introduces you to how lists, values, functions, integers, floats, basic operators, references, and arrays work in OCaml.
```

While it's often beneficial to put what we'll be learning at the front of a tutorial/lesson, Thibaut felt this list was too long, and it might discourage/overwhelm the reader. This is my suggestion for edits.

Also, Thibaut suggested we put the "prerequisites" at the top and the "next up" at the bottom as parts of the structure rather than the text. For example, the bottom of each page will have a link to what's next on the right-hand side and a link to the one before on the left.

```should we put in a link to where they can learn code? I don't think it's necessary, perhaps. They can Google for sure, unless we'll have a basic intro somewhere we can link to in the future.```